academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md

```diff
@@ -25,7 +25,7 @@ MAKES requires data it indexes to be placed in a single JSON file, with each lin
 
    ![Azure Data Lake Analytics - New job](../graph/media/samples-azure-data-lake-hindex/new-job.png "Azure Data Lake Analytics - New job")
 
-1. Copy the code contained in samples/CreateFunctions.usql under the MAG release and paste it into the script window.
+1. Copy the code from in samples/CreateFunctions.usql under the MAG release and paste it into the script window.
```

Comment:
Instead of “Copy code from sample/CreateFunctions.usql”, please change to “download” and “open file”. See “Define functions to extract MAG data" in MAG document.